website/docs/r/accessanalyzer_analyzer.html.markdown

```diff
@@ -27,7 +44,7 @@ The following arguments are required:
 The following arguments are optional:
 
 * `tags` - (Optional) Key-value map of resource tags.
-* `type` - (Optional) Type of Analyzer. Valid value is currently only `ACCOUNT`. Defaults to `ACCOUNT`.
+* `type` - (Optional) Type of Analyzer. Valid values are `ACCOUNT` or `ORGANIZATION`. Defaults to `ACCOUNT`.
```

Comment:
how do we feel about adding `Forces new resource` to `(Optional)` in docs? i've seen it in some places but not strictly everywhere where applicable. this could be a good place since the schema has changed for this attribute but also could apply to the `analyzer_name` argument

Reply:
I'd say the convention to exclude that information in the resource documentation is more typical for two reasons:

* The `terraform plan/apply` output authoritatively shows when and where resource recreation behavior occurs (except of course the TypeSet weirdness we were looking at earlier 😬 )
* Previous changes to remove `ForceNew` and support in-place updates where it was documented tended to be forgotten since the information is atypical and changes more frequently than `Optional` vs `Required`, so quite a few followup documentation PRs were necessary to bring things up to date

That's not to say we shouldn't do it -- just some insight on the subject. When the provider documentation is more automatically generated based on the schema, I think that will be a good inflection point on whether or not its worth including more holistically across all resources, since then the change will theoretically be trivial.